README.md

```diff
@@ -0,0 +1,26 @@
+# CSV & Enumerables Practice
+
+In this exercise you are asked to process a CSV file with Olymic Games data.  This exercise has two parts.  The first allowing you to practice reading in data from a CSV file, processing it and writing the results back to a file.  The second involves using Enumerable methods to manipulate the data from the file.
```

Comment:
"Olymic Games" -> "Olympic Games" typo

Reply:
Grammar: "The first allowing you to ..." -> "The first part allows you to" or something similar, like, "In the first part, you will practice reading in data ..."